backups/omarchy-dotfiles-20260214-111647/.config/omarchy/themes/aether/waybar.css

```
@@ -0,0 +1,2 @@
+@define-color background #0a100b;
```

Biome (2.3.14)

[error] Unexpected unknown at-rule: define-color 

define-color is not a standard CSS at-rule, which may lead to unexpected styling results or failure to interpret the styles as intended.
See MDN web docs for a known list of at-rules.
To fix this issue, consider removing the unknown at-rule.

(lint/suspicious/noUnknownAtRules)